cool-core/src/main/java/com/nus/cool/core/cohort/refactor/filter/RangeFilter.java

```diff
@@ -87,7 +87,7 @@ public BitSet accept(String[] values) throws RuntimeException {
   @Override
   public boolean accept(Scope scope) throws RuntimeException {
     for (Scope u : acceptRangeList) {
-      if (u.isSubset(scope)) {
+      if (u.isSubset(scope) || u.isIntersection(scope)) {
```

Comment:
Error.
Filter [10,19) and input value [9,14) are intersection.
But Filter can not accept the [9, 14)